graphql/database/experiments/features.js

```diff
@@ -135,6 +136,66 @@ const features = {
       },
     ],
   },
+  [YAHOO_SEARCH_NEW_USERS_V2]: {
+    defaultValue: 'Tooltip',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Control',
+            $exists: true,
+          },
+        },
+        force: 'Control',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Tooltip',
+            $exists: true,
+          },
+        },
+        force: 'Tooltip',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Notification',
+            $exists: true,
+          },
+        },
+        force: 'Notification',
+      },
+      /* End internal overrides */
+      {
+        variations: ['Control', 'Tooltip', 'Notification'],
+        weights: [0.34, 0.33, 0.33],
+        coverage: 1.0,
+        condition: {
+          isTabTeamMember: {
+            $eq: true,
+          },
+          joined: {
+            // todo, update gt time when launching. also end previous experiment
```

Comment:
This should work to update both experiments at the same time. You could also set this experiment to exclude anybody in the previous experiment.

Reply:
I'm not sure I follow this. Also, any reason why we can't merge without these dates being set if the coverage for v2 is 0?

Reply:
I feel like it'd be safe to just merge in both the date values when we actually turn on the coverage and remove the `isTabTeamMember` condition.

Reply:
That totally works. My comment about setting up an exclusion would be just an extra safeguard in this case and not necessary. We'd want it only if there's some reason to decouple ending the first experiment from launching the second.